Sources/UserModuleKit/Interfaces/Role+Interface.swift

```diff
@@ -7,36 +7,36 @@
 
 import FeatherModuleKit
 
-public protocol UserRoleInterface {
+public protocol UserRoleInterface: Sendable {
 
     func list(
         _ input: User.Role.List.Query
     ) async throws -> User.Role.List
 
     func reference(
-        keys: [ID<User.Role>]
+        ids: [ID<User.Role>]
     ) async throws -> [User.Role.Reference]
 
     func create(
         _ input: User.Role.Create
     ) async throws -> User.Role.Detail
 
     func get(
```

Comment:
It should be renamed to "require" because of Detail return is not optional (not Detail?)